warehouse/bigquery/bigquery.go

```diff
@@ -111,11 +111,20 @@ func (bq *HandleT) DeleteTable(tableName string) (err error) {
 }
 
 func (bq *HandleT) CreateTable(tableName string, columnMap map[string]string) (err error) {
+
+	primaryKey := "id"
+	if column, ok := primaryKeyMap[tableName]; ok {
+		primaryKey = column
+	}
+	clusteringColumns := strings.Split(strings.TrimSpace(primaryKey), ",")
 	pkgLogger.Infof("BQ: Creating table: %s in bigquery dataset: %s in project: %s", tableName, bq.Namespace, bq.ProjectID)
 	sampleSchema := getTableSchema(columnMap)
 	metaData := &bigquery.TableMetadata{
 		Schema:           sampleSchema,
 		TimePartitioning: &bigquery.TimePartitioning{},
+		Clustering: &bigquery.Clustering{
```

Comment:
Add env variable say `Warehouse.bigquery.enableClustering` to create clustered tables